### DISCUSSION TITLE
Control tower library

### DISCUSSION THREAD

**Discussion by costastf** [superwerker/superwerker#168] [General]
Hi @s0enke a while back you created an issue on one of the libraries that I have created for aws authentication https://github.com/schubergphilis/awsauthenticationlib and I mentioned that i was working on a control tower library (https://github.com/schubergphilis/controltowerlib) and that I would have a look at your project. I have not played with your project yet, I am going to do it next week but it does look like something that I would love to use. I wanted to let you know that i have open sourced my controltowerlib that can manage control tower almost entirely and actually today I have implemented deployment of control tower and I think that maybe we could work together since we are trying to achieve the same thing, but have focused on different aspects of it. Looking forward to your feedback and thoughts! Keep up the amazing work, and next week i will dive nose deep into your project!

**Comment by s0enke** [comment]
Ho @costastf , sorry for not getting back sooner to you. I've seen you Control Tower lib, awesome work!

Using it to setup or update CT would be a great addition to superwerker since we currently rely on Synthetics/Puppeteer which is rather fragile (it already broke once here: https://github.com/superwerker/superwerker/issues/162). The [code to set up CT seems to be commented out](https://github.com/schubergphilis/controltowerlib/blob/6f08d86f8cfc0c6e97959a8ec78152b21a28be0f/controltowerlib/controltowerlib.py#L1396-L1444), or did I overlook something? 
Edit: Found the [PR here](https://github.com/schubergphilis/controltowerlib/pull/6)

Regarding testing out superwerker, feel free to contact us here, via Slack. We can also have a call, feel free to contact me at soenke DOT ruempler AT superluminar DOT io :)
